datastore-design/go-grpc-demo/examples/clients/python/client_knmi_param.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# tested with Python 3.11
+import concurrent
+import os
+from multiprocessing import cpu_count
+from pathlib import Path
+from time import perf_counter
+from typing import List
+from typing import Tuple
+
+import datastore_pb2 as dstore
+import datastore_pb2_grpc as dstore_grpc
+import grpc
+import pandas as pd
+import xarray as xr
+from parameters import knmi_parameter_names
+from google.protobuf.timestamp_pb2 import Timestamp
+
+
+def netcdf_file_to_requests(file_path: Path | str) -> Tuple[List, List]:
+    time_series_request_messages = []
+    observation_request_messages = []
+    # TODO: How to deal with IDs. At the moment, I set them manually, but if the database or server could handle it,
+    #   it would help when going for parallel processing when inserting. Do we want to use a UUID?
```

Comment:
I don't think neither the DB nor the gRPC server should provide any automatic generation of IDs. Not even unofficially for the purpose of testing. A fundamental premise of the data store service is (still) that IDs need to managed solely by external clients. The service allows you to store them and refer to them, that's all. Providing an ID-generation feature could make it tempting for external clients (other than our own test clients) to start to use that, and eventually it could become a demand for adding even more utility features to the data store (like translation between metadata formats). This whole subject probably needs to be discussed at a higher level, but it is good to be conscious about it already now.

Reply:
I forgot to remove this comment, it was a note so that I would bring it up in our meeting. As we spoke about it, and we agreed on what you propose above. I will remove this TODO comment.

As the data inserted is for testing would it be okay to leave the `ts_id` index as it is in there?

Reply:
Sure! I usually have very few opinions about internals in a test program. The ts_id index seems very sensible.